tools/metadata/test/main.nf

```diff
@@ -10,7 +10,7 @@ log.info ("Starting tests for metadata...")
 /* Module inclusions
 --------------------------------------------------------------------------------------*/
 
-include {fastq_metadata as meta_se; fastq_metadata as meta_pe;} from '../main.nf'
+include {fastq_metadata as meta_se; fastq_metadata as meta_pe; smartseq2_fastq_metadata as meta_smartseq} from '../main.nf'
```

Comment:
you dont need to use an alias here - i only used b4 because i am importing the process twice